### ISSUE TITLE
Lack of web compatibility blocks ObjectScript 2.12+ extension from being usable in the browser version of VS Code

### ISSUE THREAD

**Issue by gjsjohnmurray** [intersystems-community/intersystems-servermanager#232]
See https://community.intersystems.com/post/vscode-objectscript-github#comment-266991

**Comment by isc-bsaviano** [comment]
The browser version of vscode-objectscript does almost nothing:

https://github.com/intersystems-community/vscode-objectscript/blob/ef66f3abac696dde8fd04c5b947dbabc41bbb7dc/src/web-extension.ts#L14-L31

**Comment by nickmitchko** [comment]
This **_should_** work in VSCodium and other "browser based" vscodes.

**Comment by nickmitchko** [comment]
@isc-bsaviano Just edited, code-server is where I am running with success. I guess the original issue is vscode.dev?

**Comment by isc-bsaviano** [comment]
Yes, vscode.dev is a browser-only environment where this extension isn't supported.
